device/ipv6.yaml

```diff
@@ -37,3 +35,33 @@ components:
           type: string
           format: ipv6
           default: ::0
+
+    # Can not derive LoopbackIpv6 from Ipv6Base as default prefix length must be 128.
+    Interface.LoopbackIpv6:
+      x-include: 
+      - '../common/common.yaml#/components/schemas/Named.Object'
+      description: >-
+        An IPv6 loopback interface
+      type: object
+      required: [interface_name]
+      properties:
+        interface_name: 
```

Comment:
should this be ethernet_name instead or will the x-constraint be extended in the future?

Reply:
Yes, similar to IPv4 Loopback.